metadata-mapper/metadata_mapper/lambda_function.py

```diff
@@ -14,6 +15,8 @@ def get_source_vernacular(source_type):
         return NuxeoVernacular
     if source_type == 'oac_dc':
         return OAC_Vernacular
+    if source_type == 'islandora_oai_dc_mapper':
```

Comment:
I found this function, `dir()` and I wonder if it, or something similar would be useful in eliminating this registration step. https://github.com/ucldc/rikolti/blob/enrichments/metadata-mapper/lambda_function.py#L27